structs.go

```diff
@@ -958,7 +958,7 @@ type GuildTemplate struct {
 	Description string `json:"description"`
```

Comment:
`omitempty` because:

![image](https://user-images.githubusercontent.com/42876964/171845783-affaa814-4076-4821-96a0-119f43568f80.png)

`*string` because [Modify Guild Template](https://discord.com/developers/docs/resources/guild-template#modify-guild-template-json-params) requires it:

![image](https://user-images.githubusercontent.com/42876964/171846199-1937804a-8520-4307-9d18-c51c6915613f.png)

Can't use just `omitempty`, because it would be impossible to set the description to empty.

```suggestion
	Description *string `json:"description,omitempty"`
```
---

P. S. Same goes for the `name`, to work correctly in "Modify Guild Template" it should be a pointer.